go.mod

```diff
@@ -1,24 +1,18 @@
 module github.com/codeready-toolchain/host-operator
 
 require (
-	cloud.google.com/go v0.40.0 // indirect
-	github.com/Azure/go-autorest v13.0.0+incompatible // indirect
 	github.com/appscode/jsonpatch v0.0.0-20190625103638-320dcdd0e1f7 // indirect
-	github.com/codeready-toolchain/api v0.0.0-20190813211537-c2a9a838ce38
-	github.com/codeready-toolchain/toolchain-common v0.0.0-20190830131634-231d1bd08d5f
+	github.com/codeready-toolchain/api v0.0.0-20190906003527-443e8ceac134
+	github.com/codeready-toolchain/toolchain-common v0.0.0-20190904115454-79e74322be0a
+	github.com/codeready-toolchain/toolchain-e2e v0.0.0-20190904123258-c90f69f65a94 // indirect
```

Comment:
This shouldn't be here at all

Reply:
This was cleaned up once I ran go mod tidy.